src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+import model.Ball;
+
+public class Lottery {
+    private final Random rand = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Don't create a new instance of `ColorSupplier` every time the `getRandomBall` method is called. Instead, you can create a single instance as a class-level field and reuse it, similar to how `Random` is used.

Reply:
Useful point, fix this